client/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterClient.java

```diff
@@ -0,0 +1,378 @@
+import org.apache.servicecomb.service.center.client.model.MicroservicesResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Created by   on 2019/10/16.
+ */
+public class ServiceCenterClient {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ServiceCenterClient.class);
+
+    private ServiceCenterRawClient httpClient;
+
+    /**
+     * Use default config parameter
+     */
+    public ServiceCenterClient() {
+        this(new ServiceCenterRawClient());
+    }
+
+    /**
+     * Add TLS config of client
+     * @param tlsConfig
+     */
+    public ServiceCenterClient(TLSConfig tlsConfig){
+        this(tlsConfig,null);
+    }
+
+    /**
+     * Add extraGlobalHeaders to http request
+     * @param extraGlobalHeaders
+     */
+    public ServiceCenterClient(Map<String, String> extraGlobalHeaders) {
+        this(null,0,null,null,extraGlobalHeaders);
+    }
+
+    /**
+     * Add TLS config and extraGlobalHeaders
+     * @param tlsConfig
+     * @param extraGlobalHeaders
+     */
+    public ServiceCenterClient(TLSConfig tlsConfig,Map<String, String> extraGlobalHeaders){
+        this(null,0,null,tlsConfig,extraGlobalHeaders);
+    }
+
+    /**
+     * Customized host,port,
+     * @param host
+     * @param port
+     */
+    public ServiceCenterClient(String host, int port) {
+        this(host, port, null,null, null);
+    }
+
+    /**
+     * Customized host,port,domainName,TLSConf, headers and any one parameter can be null.
+     * @param host
+     * @param port
+     * @param domainName
+     * @param tlsConfig
+     * @param extraGlobalHeaders
+     */
+    public ServiceCenterClient(String host, int port, String domainName, TLSConfig tlsConfig,  Map<String, String> extraGlobalHeaders) {
+        HttpTransport httpTransport = HttpTransportFactory.getDefaultHttpTransport();
+        if(tlsConfig!=null){
+            httpTransport = new TLSHttpsTransport(tlsConfig);
+        }
+        httpTransport.addHeaders(extraGlobalHeaders);
+        this.httpClient = new ServiceCenterRawClient.Builder().setHost(host).setPort(port).setDomainName(domainName).setHttpTransport(httpTransport).build();
+    }
+
+    public ServiceCenterClient(ServiceCenterRawClient serviceCenterRawClient) {
+        this.httpClient = serviceCenterRawClient;
+    }
+
+    /**
+     * Get service-center instances message
+     *
+     * @return MicroserviceInstancesResponse; when error happens,return null
+     */
+    public MicroserviceInstancesResponse getServiceCenterInstances() {
+        try {
+            HttpResponse response = httpClient.getHttpRequest("/registry/health", null, null);
+            if (response.getStatusCode() == HttpStatus.SC_OK) {
+                LOGGER.info("getServiceCenterInstances result = " + response.getContent());
+//                return JSON.parseObject(response.getContent(), MicroserviceInstancesResponse.class);

```

Comment:
health? if this is wrong? And do not log successful response.

Reply:
thanks for reminding, the problem is fixed now.